src/storage/worditem-indexeddb-driver.js

```diff
@@ -437,4 +437,43 @@ static get currentDate () {
     }
     return wordItem
   }
+
+  getByStorageID (dataItems, ID) {
+    return dataItems.find(item => this.makeIDCompareWithRemote(item) === ID)
+  }
+
+  comparePartly (changeItem, sourceItem) {
+    let part = 'context'
+    if (!sourceItem[part]) {
+      return null
+    }
+    // console.info('****************comparePartly', changeItem, sourceItem)
+    if (sourceItem[part] && !changeItem[part]) {
+      changeItem[part] = sourceItem[part]
+      return changeItem
+    }
+    if (sourceItem[part] && changeItem[part]) {
+      changeItem = this.mergeContextData(changeItem, sourceItem)
+      return changeItem
+    }
+  }
+
+  mergeContextData (changeItem, sourceItem) {
```

Comment:
please add method/param description comments

Reply:
This method was removed